### PR TITLE
Fix inteminent sks keyserver failure

### PR DESCRIPTION
**Pull request by ProfFan**
Hope CI works

### CODE REVIEW

**Comment by johnwlambert** [comment]
Thanks @ProfFan . Any idea why 2 of the checks were cancelled?

File: .github/workflows/build-linux.yml [hunk 1]
```diff
@@ -53,8 +53,9 @@ jobs:
             # pool.sks-keyservers.net is the SKS GPG global keyserver pool
             # 15CF4D18AF4F7421 is the GPG key for the LLVM apt repository
             # This key is not in the keystore by default for Ubuntu so we need to add it.
-            gpg --keyserver pool.sks-keyservers.net --recv-key 15CF4D18AF4F7421
-            gpg -a --export 15CF4D18AF4F7421 | sudo apt-key add -
+            LLVM_KEY=15CF4D18AF4F7421
+            gpg --keyserver ipv4.pool.sks-keyservers.net --recv-key $LLVM_KEY || gpg --keyserver ha.pool.sks-keyservers.net --recv-key $LLVM_KEY
```
Comment (johnwlambert) [inline .github/workflows/build-linux.yml:57]: Thanks for this update Fan. Do you mind adding a brief note about what ipv4 represents, vs. the pool we used beforehand?

**Review by johnwlambert — CHANGES_REQUESTED** [review]

**Comment by johnwlambert** [comment]
Will they time out for all future branches, as well? Can we increase the timeout length, or should we remove those 2 from the CI?

**Comment by ProfFan** [comment]
This is not related to the macOS timeouts, GitHub uses MacStadium for the macOS CI, and they will terminate your long-running jobs at random.

File: .github/workflows/build-linux.yml (hunk 1 above)
Comment (ProfFan) [reply .github/workflows/build-linux.yml:58]: done

**Review by johnwlambert — APPROVED** [review]

**Pull request merged**